src/components/Dashboard/FilingHistoryList/FutureEffectivePending.vue

```diff
@@ -39,21 +42,18 @@ export default class FutureEffectivePending extends Mixins(DateMixin) {
     if (this.filing.isFutureEffectiveIaPending) {
       return {
         subtitle: 'Incorporation Pending',
-        filingLabel: 'incorporation',
-        companyLabel: (this.getEntityName || 'this Numbered Benefit Company')
```

Comment:
`ifFutureEffectiveIaPending` can be True for Coop IAs, so I changed "this Numbered Benefit Company" to "this company" and moved it into the template (same as FutureEffective.vue).